spec/data/enums_handler.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
+# frozen_string_literal: true
+
+# An enum representing card suits
+class Suit < T::Enum
+  enums do
+    # The spades suit
+    # @see https://en.wikipedia.org/wiki/Spades_(suit)
+    Spades = new
+    # The hearts suit
+    Hearts = new
+    Clubs = new('Clubs')
+    diamonds_serialized_value = 'Diamonds'
+    Diamonds = new(diamonds_serialized_value)
```

Comment:
This will appear as `new(diamonds_serialized_value)` in the documentation, with no other reference to `diamonds_serialized_value` for clarity. I don't love that, but I can live with it. LMK if you have any thoughts.

Reply:
I'm okay with it either way, I don't think this case is that important since people shouldn't be adding extra logic such as local variable assignments to `enums` blocks.

Reply:
This turns out to be consistent with YARD, if you write something pathological like:
```
class A
  b = 1
  C = b
end
```
so I think this is 🆗